src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+
+class TonMacronFriendInvitationAdmin extends AbstractAdmin
+{
```

Comment:
I think it's better to order by step number first and sequence (content key) number then. Thus we can disable pagination for such a list. This way we could override the list template to have an output similar to the following one:

````
Étape 0 : choix hors étapes du tunnel Ton Macron
---------------------------------------------------------

Label | Contenu
------------ | -------------
Introduction | Contenu de l'introduction
Conclusion | Contenu de la conclusion

Étape 1 : situation personnelle de l'ami
---------------------------------------------

Label | Contenu
------------ | -------------
A | Contenu de A
B | Contenu de B
C | Contenu de C

Étape 2 : projet de l'ami
---------------------------

Label | Contenu
------------ | -------------
D | Contenu de D
E | Contenu de E
F | Contenu de F
```